src/packet/rustq/src/lib.rs

```diff
@@ -96,7 +96,7 @@ impl WrapperPacketQueue {
                 tracing::trace!(dport = ?p.dport(), "enqueueing packet");
                 match match &mut self.inner {
                     WrapperPacketQueueInner::ClassTokenBucket(q) => q.0.enq(p),
-                    WrapperPacketQueueInner::DeficitRoundRobin(q) => q.0.enq(p),
+                    WrapperPacketQueueInner::DeficitRoundRobin(q) => {q.0.dbg(Duration::ZERO); q.0.enq(p)},
```

Comment:
we probably don't want to call `dbg` *every* time a packet is enqueued, and it would probably be useful to call `dbg` in all cases, not just the DRR case.

Reply:
there are multiple possible ways to log more occasionally:
- based on a timer
- based on an RNG
- "every X packets"
- some combination of the above, e.g., "every X packets, but also no more than 1 second apart"

Reply:
also, what is `Duration::ZERO` doing here? We should probably instead pass in the elapsed time between logs.